lib/default_conf.js

```diff
@@ -1,4 +1,5 @@
-let {version} = require('../index')
+const { version } = require('../package.json')
+
```

Comment:
Useless new empty line (should be linted too)

Reply:
The eslint-peopledoc-config has no rule for this 🤔 . Personally I like having a empty line between import/require and the rest of the code for readability. Maybe we can ask the tribe waht they prefer

Reply:
I agree for one empty line, not 2 ^^

Reply:
Oh yes there's two newline 😅